libs/wingc/src/jsify/code.rs

```diff
@@ -0,0 +1,126 @@
+#[derive(Default)]
+pub struct CodeMaker {
+	lines: Vec<(usize, String)>,
+	indent: usize,
+}
+
+impl CodeMaker {
+	pub fn open<S: Into<String>>(&mut self, line: S) {
```

Comment:
It's useful to also support open_block() and close_block()

Reply:
Let's keep the API minimal for now 🙏